src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
+        return Files.readAllLines(Paths.get(filePath));
+    }
+}
```

Comment:
According to the checklist, you should not keep all logic in a single package. Consider creating a separate package for services like FileReader, such as `service` or `service.impl`. This will help organize your code better and adhere to good package structuring practices.